src/main/java/com/oltpbenchmark/benchmarks/templated/TemplatedBenchmark.java

```diff
@@ -170,38 +181,44 @@ private CustomClassLoader loadQueryTemplates(String file) {
             JAXBElement<TemplatesType> result = unmarshaller.unmarshal(streamSource, TemplatesType.class);
             TemplatesType templates = result.getValue();
 
-            for (TemplateType template : templates.getTemplateList()) {
+            for (TemplateType template : templates.getTemplate()) {
```

Comment:
Why not `getTemplateList()` anymore?

Reply:
All of these files were automatically generated but I will make sure to change it :)

Reply:
I was just looking for an explanation actually.  I don't think you should go editing auto-generated code after that fact.    That leads to a maintenance nightmare.  Please revert it to whatever the standard auto generated code was for now.

Even better would be that the auto-generated code gets auto-generated during build time without extra steps so it can automatically rev with dependencies and we know when there are issues, but I think that's a separate issue (#365).